app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class UserController < ApiController
+      def update
+        if current_user.id == params[:id].to_i
+          current_user.update(user_params)
+          render json: current_user, status: :ok
+        else
+          render_unauthorized
+        end
+      end
+
+      private
+
+      def user_params
+        params.permit(:name, :lastname, :gender)
```

Comment:
Please use snake_case, maybe you'll have to rename the DB column
```suggestion
        params.permit(:name, :last_name, :gender)
```

Reply:
you also will need to update the model and migration related